bom/application/pom.xml

```diff
@@ -120,9 +120,9 @@
         <slf4j-jboss-logmanager.version>2.0.0.Final</slf4j-jboss-logmanager.version>
         <wildfly-common.version>1.7.0.Final</wildfly-common.version>
         <wildfly-client-config.version>1.0.1.Final</wildfly-client-config.version>
-        <wildfly-elytron.version>2.2.3.Final</wildfly-elytron.version>
-        <jboss-threads.version>3.5.1.Final</jboss-threads.version>
-        <vertx.version>4.5.4</vertx.version>
+        <wildfly-elytron.version>2.3.1.Final</wildfly-elytron.version>
+        <jboss-threads.version>3.6.0.Final</jboss-threads.version>
```

Comment:
Are these upgrades really necessary? They are not in my PR but I'm not sure they are?